core/api/tests/test_business_plans.py

```diff
@@ -121,139 +116,118 @@ def test_list_year_filter(self, user, _setup_bp_list):
         assert all(bp["year_end"] == 2023 for bp in response.json())
 
 
-@pytest.fixture(name="_setup_new_business_plan_create")
-def setup_new_business_plan_create():
-    return {
-        "name": "Test BP",
-        "year_start": 2020,
-        "year_end": 2023,
-        "status": "Endorsed",
-    }
-
-
-class TestBPCreate:
+class TestBPImportValidate:
```

Comment:
please add a test with an invalid file structure and a file with a wrong country / agency